spec/transition_spec.rb

```diff
@@ -0,0 +1,40 @@
+describe ProMotion::Menu::Transition do
+
+  before do
+    @object = Object.new
+    @object.extend(ProMotion::Menu::Transition)
+
+    @delegate = UIApplication.sharedApplication.delegate
+    @left = LeftNavScreen.new
+    @right = RightNavScreen.new
+    @content = BlankScreen.new
+  end
+
+
+  describe "#visualStates" do
```

Comment:
Yeah, I had some trouble writing these specs, but this is a good suggestion. I'm on East Coast time so I'll take a whack at re-writing this spec tomorrow.